apps/test/unit/templates/teacherDashboard/teacherSectionsReduxTest.js

```diff
@@ -662,6 +670,34 @@ describe('teacherSectionsRedux', () => {
       state = reducer(state, editSectionProperties({scriptId: 37}));
       expect(state.sectionBeingEdited.stageExtras).to.equal(true);
     });
+
+    it('when updating script assignment for a section, ttsAutoplayEnabled defaults to false', () => {
+      let state = reducer(
+        editingNewSectionState,
+        setPreReaderScriptIds(preReaderScripts)
+      );
+      state = reducer(state, editSectionProperties({scriptId: 2}));
+      expect(state.sectionBeingEdited.ttsAutoplayEnabled).to.equal(false);
+
+      state = reducer(state, editSectionProperties({scriptId: 37}));
+      expect(state.sectionBeingEdited.ttsAutoplayEnabled).to.equal(false);
+    });
+
+    // TODO: add this test when tts autoplay is enabled by default for pre-reader scripts
```

Comment:
NIT: similar to my previous comment, it would be cleaner to have this test skipped for now (`it.skip('switching...`) instead of commented out